index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    
+    <div class="container">
+    
+    <header class="portfolio-header">
+    <div class="logo">My Logo</div>
+    
+        <div class="menu-mobile">
+        <div class="menu-bar"></div>
+        <div class="menu-bar"></div>
+        <div class="menu-bar"></div>
+    </div>
+
+    </header> 
+  
+    
+
+<h1 class="salutation">
+   Hello. I’m Austin.
+    Glad to see you!</h1>
```

Comment:
As mentioned by the previous reviewer, please style your main title so that it is larger and spans three lines instead of two. You can consider placing a `<br>` tag between the text to achieve this effect
```suggestion
<h1 class="salutation">
   Hello. I’m Austin. <br>
    Glad to see you!</h1>
```